docs/content/docs/connectors/datastream/cassandra.md

```diff
@@ -37,7 +37,7 @@ To use this connector, add the following dependency to your project:
 
 {{< artifact flink-connector-cassandra withScalaVersion >}}
 
-Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{< ref "docs/dev/datastream/project-configuration" >}}).
+Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{< ref "docs/dev/configuration" >}}).
```

Comment:
NB: All these links currently send me to http://localhost:1313/flink/flink-docs-master/docs/dev/configuration/. That page is blank, it should be http://localhost:1313/flink/flink-docs-master/docs/dev/configuration/overview/ I think. So:

```suggestion
Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{< ref "docs/dev/configuration/overview" >}}).
```